File: src/main/java/nu/peg/slack/pt/api/transport/model/Connection.java
package nu.peg.slack.pt.api.transport.model;

import java.util.List;

import lombok.Data;

@Data
public class Connection {

    private Checkpoint from, to;
    private String duration;
    private Service service;
    private List<String> products;
    private Integer capacity1st, capacity2nd;
    private List<Section> sections;

}

File: src/main/kotlin/nu/peg/slack/pt/api/transport/model/Connection.kt
package nu.peg.slack.pt.api.transport.model

import lombok.Data

@Data
class Connection {

    private val from: Checkpoint? = null
    private val to: Checkpoint? = null
    private val duration: String? = null
    private val service: Service? = null
    private val products: List<String>? = null
    private val capacity1st: Int? = null
    private val capacity2nd: Int? = null
    private val sections: List<Section>? = null

}
